plugins/inputs/syslog-ng/syslogng_test.go

```diff
@@ -0,0 +1,66 @@
+package syslogng
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func SyslogNgCtlRunnerTest(output string) func(string, config.Duration, bool) (*bytes.Buffer, error) {
+	return func(string, config.Duration, bool) (*bytes.Buffer, error) {
+		return bytes.NewBuffer([]byte(output)), nil
+	}
+}
+
+func TestParseFullOutput(t *testing.T) {
+	acc := &testutil.Accumulator{}
+	v := &SyslogNg{
+		run: SyslogNgCtlRunnerTest(fullOutput),
+	}
+	err := v.Gather(acc)
+
+	require.NoError(t, err)
+	require.True(t, acc.HasMeasurement("syslog-ng"))
+
+	require.Len(t, acc.Metrics, 26)
+	require.Equal(t, 26, acc.NFields())
+
+	acc.AssertContainsFields(t, "syslog-ng", parsedFullOutput)
```

Comment:
How about
```suggestion
	require.NoError(t, v.Gather(acc))
	
	require.True(t, acc.HasMeasurement("syslog-ng"))
	require.Len(t, acc.Metrics, 26)
	require.Equal(t, 26, acc.NFields())
	acc.AssertContainsFields(t, "syslog-ng", parsedFullOutput)
```
It's only spaces, but it's easier for the reader to understand what belongs together, IMO.